22_to_do_list/app.js

```diff
@@ -0,0 +1,87 @@
+//Selectores
+
+const inputText = document.getElementById("task")
+const outputText = document.querySelector(".js-value-output")
+const addBtn = document.querySelector(".js-btn-add")
+const list = document.querySelector(".js-list")
+const alert = document.querySelector(".js-alert")
+const clear = document.querySelector(".js-btn-clear")
```

Comment:
Well done using `.js-` prefixes! Do you feel like they help you to organize the code?